eZ/Publish/Core/Repository/Values/ObjectState/ObjectState.php

```diff
@@ -26,16 +23,33 @@
  */
 class ObjectState extends APIObjectState
 {
-    use MultiLanguageTrait;
-    use MultiLanguageNameTrait;
-    use MultiLanguageDescriptionTrait;
```

Comment:
Note: Actually we can't use these traits here because, as mentioned above, `mainLanguageCode` is called here `defaultLanguageCode`